### DISCUSSION TITLE
New crazy ideas with banks

### DISCUSSION THREAD

**Discussion by santta** [tttapa/Control-Surface#503] [Show and tell]
Hi, I keep experimenting with my custom foot-controller. And new strange ideas come to me)))))
I got the idea to use it with different devices that accept CC and other PC devices.

Is it possible to implement switching from CCButton to PCButton, the same buttons, through the bank selector?

**Comment by tttapa** [comment]
You could technically do it by using `no_address` for some of the banks, as shown here: https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d79/One-Pot-Both-PB-and-CC_8ino-example.html

However, it might be easier to write a [Custom-MIDI-Output-Element](https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d3b/Custom-MIDI-Output-Element_8ino-example.html) or a [Custom-Output-Element-Bankable](https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d6b/Custom-MIDI-Output-Element-Bankable_8ino-example.html) as they allow for more flexibility.

**Reply by santta** [reply]
thank you, Pieter,  I will try and if I can implement it, I will definitely share it with a sketch))
